release-notes/release-notes/5.7-release-notes.md

```diff
@@ -347,7 +347,10 @@ We implemented Istio mTLS support to ensure secure/TLS in-cluster communication
 - We fixed a bug that was stripping query parameters from Relative URLs if they were being used as the Application URL of the applets. In the below screenshot, the bug would have stripped out the "q=1" parameter, leaving only an Absolute URL in lieu of a Relative one.
 - We fixed an issue with the behaviour of cached calculated columns when using multi-select data types (Link, Choice, and Hierarchy) with **Change Approval** enabled. These data types should now work as expected.
 
-<figure><img src="../../.gitbook/assets/image (175).png" alt=""><figcaption></figcaption></figure>
+  <figure><img src="../../.gitbook/assets/image (175).png" alt=""><figcaption></figcaption></figure>
+
+- We fixed an issue where the use of a CASE statement on a Link reference with a condition caused an incorrect query.
```

Comment:
• We improved query performance when using a CASE statement on a link reference
• 